OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -424,6 +424,16 @@ public virtual int GetBuildTime(ActorInfo unit, BuildableInfo bi)
 			return time;
 		}
 
+		protected void PauseProduction(string itemName, bool paused)
+		{
+			var item = Queue.FirstOrDefault(a => a.Item == itemName);
+
+			if (item != null)
+			{
```

Comment:
Unnecessary `{``}`.  Please also remove the newline between the definition and the conditional.